internal/cli/atlas/livemigration/link/create.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package link
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var createTemplate = "Link Token '{{.LinkToken}}' successfully created.\n"
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store        store.LinkTokenCreator
+	accessListIP []string
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	createRequest := opts.newTokenCreateRequest()
+
+	r, err := opts.store.CreateLinkToken(opts.OrgID, createRequest)
```

Comment:
so people can rely on the ENV or profile set org ID
```suggestion
	r, err := opts.store.CreateLinkToken(opts.ConfigOrgID(), createRequest)
```